table/column_test.go

```diff
@@ -450,25 +460,25 @@ func (t *testTableSuite) TestGetDefaultValue(c *C) {
 		ctx.GetSessionVars().StmtCtx.BadNullAsWarning = !tt.strict
 		val, err := GetColDefaultValue(ctx, tt.colInfo)
 		if err != nil {
-			c.Assert(tt.err, NotNil, Commentf("%v", err))
+			require.Errorf(t, tt.err, "%v", err)
```

Comment:
weird, shall we assert error message equal?

cc @coocood

Reply:
With an offline discussion with @coocood we agree that postpone the problem later and focus on migration in this PR.

Reply:
:ok_hand: